Lib/zipfile.py

```diff
@@ -1783,23 +1784,6 @@ def write(self, filename, arcname=None,
             else:
                 zinfo._compresslevel = self.compresslevel
 
-        if zinfo.is_dir():
```

Comment:
Does this mean that `write` can no longer be used to create a directory?  I suspect that will break existing code.

Reply:
I think write still accepts directory zinfo, now it’s handled on line 1775 by calling mkdir instead of directly inline here.